tests/unit/examples/test_serving_ranking_models_with_merlin_systems.py

```diff
@@ -105,5 +106,12 @@ def test_example_04_exporting_ranking_models(tb):
     configure_tensorflow()
     from merlin.systems.triton.utils import run_ensemble_on_tritonserver
 
-    response = run_ensemble_on_tritonserver("/tmp/data/ensemble/", outputs, batch, "ensemble_model")
+    # The schema contains int64s, while the actual data contains int32s. Not sure why.
+    schema = Schema(
+        [col_schema.with_dtype(np.int32) for col_schema in schema.column_schemas.values()]
+    )
```

Comment:
I think what's going on here is that the schema never matched the data, but the conversion from dataframe to Triton input didn't used to coerce the dtypes to schema dtypes.